[2021-10-16 10:41:03,218] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T10:38:00+00:00 [queued]>
[2021-10-16 10:41:03,250] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T10:38:00+00:00 [queued]>
[2021-10-16 10:41:03,251] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 10:41:03,251] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 10:41:03,251] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 10:41:03,294] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_0> on 2021-10-16 10:38:00+00:00
[2021-10-16 10:41:03,306] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_0', 'scheduled__2021-10-16T10:38:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpibzj1gu0', '--error-file', '/tmp/tmpzjxu_ny7']
[2021-10-16 10:41:03,307] {standard_task_runner.py:77} INFO - Job 53: Subtask get_gross_year_0
[2021-10-16 10:41:03,302] {standard_task_runner.py:52} INFO - Started process 1016 to run task
[2021-10-16 10:41:03,374] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T10:38:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 10:41:03,447] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_0
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T10:38:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T10:38:00+00:00
[2021-10-16 10:41:03,459] {python.py:152} INFO - Done. Returned value was: ['$30.15 ', '2011\n']
[2021-10-16 10:41:03,503] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_0, execution_date=20211016T103800, start_date=20211016T104103, end_date=20211016T104103
[2021-10-16 10:41:03,563] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 10:41:03,608] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
